src/test/java/org/rulez/demokracia/pdengine/showvote/VoteShowTest.java

```diff
@@ -24,176 +27,241 @@
 @TestedFeature("Manage votes")
 @TestedOperation("show vote")
 @TestedBehaviour("returns the vote in json")
-public class VoteShowTest extends CreatedDefaultVoteRegistry {
+@RunWith(MockitoJUnitRunner.class)
+@SuppressWarnings("PMD.TooManyMethods")
+public class VoteShowTest extends ShowVoteTestBase {
 
   private static final String COUNTED_ASSURANCE = "CountedAssurance";
   private static final String ADMIN_KEY = "adminKey";
-  private static final String CAN_VOTE = "canVote";
-  private static final String CAN_VIEW = "canView";
+  private static final String CAN_VOTE = "votable";
+  private static final String CAN_VIEW = "viewable";
   private static final String MIN_ENDORSEMENTS = "minEndorsements";
   private static final String COUNTED_ASSURANCES = "countedAssurances";
   private static final String NEEDED_ASSURANCES = "neededAssurances";
-  private static final String CAN_ENDORSE = "canEndorse";
+  private static final String CAN_ENDORSE = "endorsable";
   private static final String USER_NAME = "userName";
   private static final String CHOICES = "choices";
   private static final String CREATION_TIME = "creationTime";
-  private static final String CAN_ADDIN = "canAddin";
+  private static final String CAN_ADDIN = "addinable";
   private static final String VOTE_PARAMETERS = "parameters";
   private static final String NAME = "name";
   private static final String USER = "user";
   private static final String CHOICE_NAME = "choiceName";
+
   private Vote vote;
   private String voteId;
   private String adminKey;
 
-  @Override
   @Before
   public void setUp() {
-    super.setUp();
-    vote = voteManager.getVote(adminInfo.voteId);
-    vote.addChoice(CHOICE_NAME, USER);
-    voteManager.addChoice(
-        new VoteAdminInfo(adminInfo.voteId, adminInfo.adminKey), CHOICE_NAME, USER
+    vote = new Vote(
+        "votename", List.of("needed1", "needed2"), List.of("counted1", "counted2"), true, 3
     );
-    voteId = adminInfo.voteId;
-    adminKey = adminInfo.adminKey;
+    vote.addChoice(new Choice(CHOICE_NAME, USER));
+    voteId = vote.getId();
+    adminKey = vote.getAdminKey();
+
+    when(voteService.getVoteWithValidatedAdminKey(any())).thenReturn(vote);
   }
 
   @Test
   public void the_name_attribute_contains_the_name_of_the_vote() {
-    JsonObject result =
-        voteManager.showVote(new VoteAdminInfo(voteId, adminKey));
+    JsonObject result = showVoteService.showVote(
```

Comment:
This expression can be moved to a helper method and/or setUp, as used a couple of times.